packages/web/src/utils/gdpr.ts

```diff
@@ -1,14 +1,47 @@
 import { getLocation } from 'services/Location'
 
 const DISMISSED_COOKIE_BANNER_KEY = 'dismissCookieBanner'
+const IS_EU_KEY = 'isEU'
+const IS_EU_CACHE_TTL_MS = 7 * 24 * 3600 * 1000
```

Comment:
7 days feels long, no?

Reply:
I think it's fine, how often are ppl flying in and out of EU